pkg/skaffold/kubernetes/log.go

```diff
@@ -31,6 +31,26 @@ import (
 	restclient "k8s.io/client-go/rest"
 )
 
+const defaultRetry int = 5
+
+// LogAggregator aggregates the logs for all the deployed pods.
+type LogAggregator struct {
+	Muter
+
+	creationTime time.Time
+	output       io.Writer
```

Comment:
ref #228 We should think about passing some sort of OutputWriter/Logger struct here that does some nice coloring and formatting. It would be nice to reuse the same special writer in our output in run/deploy too.